numpy/testing/tests/test_utils.py

```diff
@@ -300,9 +300,9 @@ def test_nat_items(self):
             self._test_not_equal([a], b)
 
         for a, b in itertools.product(tds, dts):
-            self._test_not_equal(a, b)
+            self._assert_func(a, b)
             self._test_not_equal(a, [b])
-            self._test_not_equal([a], [b])
+            self._assert_func([a], [b])
```

Comment:
Why this change? Seems weird when the rest of the tests are all `_test_not_equal`

Reply:
reverted